src/controllers/csi/metadata/sqlite.go

```diff
@@ -308,13 +309,18 @@ func (access *SqliteAccess) GetDynakube(dynakubeName string) (*Dynakube, error)
 	var tenantUUID string
 	var latestVersion string
 	var imageDigest string
-	var maxFailedMountAttempts int
+	var maxFailedMountAttempts *int
 
 	err := access.querySimpleStatement(getDynakubeStatement, dynakubeName, &tenantUUID, &latestVersion, &imageDigest, &maxFailedMountAttempts)
 	if err != nil {
 		err = errors.WithMessagef(err, "couldn't get dynakube, name '%s'", dynakubeName)
 	}
-	return NewDynakube(dynakubeName, tenantUUID, latestVersion, imageDigest, maxFailedMountAttempts), err
+
+	if maxFailedMountAttempts == nil {
+		maxFailedMountAttempts = address.Of(-1)
```

Comment:
`maxFailedMountAttempts` should never be `nil` at this point
-1 is not a valid value

Reply:
changed from pointer to value, good catch